decidim-conferences/app/views/decidim/conferences/media/index.html.erb

```diff
@@ -9,23 +9,34 @@ edit_link(
 )
 %>
 
-<div class="row">
-  <div class="columns mediumlarge-12 conference-media">
-    <% if @media_links.any? %>
-      <section>
-        <h3 class="section-heading"><%= t("media.index.title", scope: "decidim") %></h3>
-        <p><%= t("media.index.description", scope: "decidim") %></p>
-      </section>
-      <hr class="reset mt-none mb-s">
-      <div class="media-links">
-        <ul class="list-reset">
-          <% @media_links.each do |media_link| %>
-            <%= cell "decidim/conferences/media_link", media_link %>
-          <% end %>
-        </ul>
+<%# NOTE: this page does not use a regular layout %>
+<div class="container">
+  <h1 class="h3 md:h1 decorator my-12">
+    <span class="not-sr-only"><%= t("media", scope: "layouts.decidim.conferences_nav") %></span>
+    <span class="sr-only"><%= t("media", scope: "layouts.decidim.conferences_nav") %> (<%= translated_attribute current_participatory_space.title %>)</span>
```

Comment:
:+1: 

Just regarding the comment above, this is the same fix you should make to the speakers page as well.